ui/src/literature/components/PublicNotesList.jsx

```diff
@@ -0,0 +1,30 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { List } from 'immutable';
+
+import InlineList from '../../common/components/InlineList';
+
+class PublicNotesList extends Component {
+  render() {
+    const { publicNotes } = this.props;
+    return (
+      <InlineList
+        label="Note"
+        items={publicNotes}
+        extractKey={note => note.get('value')}
+        renderItem={note => <span>{note.get('value')}</span>}
```

Comment:
could you actually make `static getValue` and use it for both `extractKey` and `renderItem`